src/main/kotlin/com/msg/gauth/domain/client/service/DeleteSeveralClientService.kt

```diff
@@ -0,0 +1,20 @@
+package com.msg.gauth.domain.client.service
+
+import com.msg.gauth.domain.client.Client
+import com.msg.gauth.domain.client.repository.ClientRepository
+import com.msg.gauth.domain.user.User
+import com.msg.gauth.domain.user.util.UserUtil
+import org.springframework.stereotype.Service
+import org.springframework.transaction.annotation.Transactional
+
+@Service
+@Transactional(rollbackFor = [Exception::class])
+class DeleteSeveralClientService(
+    private val clientRepository: ClientRepository,
+    private val userUtil: UserUtil
+) {
+    fun execute(ids: List<Long>) {
+        val user = userUtil.fetchCurrentUser()
+        ids.map { clientRepository.deleteByIdAndCreatedBy(it, user) }
```

Comment:
```suggestion
        ids.forEach { clientRepository.deleteByIdAndCreatedBy(it, user) }
```
결과 쓰는거도 없는데 map말고 forEach로 하는거 어떤가요?

Reply:
넵
https://github.com/GSM-MSG/GAuth-BackEnd/pull/173/commits/ab909cd2fb907a3798e533d7fffc0b3991abba0e